miasm2/jitter/jitcore_tcc.py

```diff
@@ -2,11 +2,11 @@
 #-*- coding:utf-8 -*-
 
 import os
-from miasm2.ir.ir2C import irblocs2C
-from subprocess import Popen, PIPE
-import miasm2.jitter.jitcore as jitcore
 from distutils.sysconfig import get_python_inc
-import Jittcc
+from subprocess import Popen, PIPE
+from miasm2.ir.ir2C import irblocs2C
```

Comment:
Actually, according to pep8 convention and in order to be homogeneous in Miasm, import order is divided between python common (as `subprocess`) and miasm import (as `miasm2. ...`) with an empty line. So, an empty line is missing here.
.